proto/nodeinstance/info/v1/info.proto

```diff
@@ -31,6 +31,9 @@ message UpdateNodeInfo {
   string machine_guid = 5;
 
   bool child = 6;
+
+  // true when child == true && parent runs ML for child
+  bool ml_on_parent = 8;
```

Comment:
Would you guys agree to maybe have a generic extra-parent-capabilities-on-the-child thing here? Something like an 'AddedValueServices` ? :-) Maybe it could be useful in the future to add more things that a parent provides for the child? (An array, string, or something).

Reply:
> Would you guys think is ok to maybe have a generic extra-parent-capabilities-on-the-child thing here?

FWIW, I think this is a must. 

>  (An array, string, or something).

I think it's better to have a separate field for each capability that the parent provides.